src/views/List.jsx

```diff
@@ -22,6 +26,35 @@ export function List({ data, listId }) {
 		);
 	};
 
+	const CopyToken = () => {
+		const [copied, setCopied] = useState(false);
+
+		const handleCopyToClipboard = () => {
+			try {
+				copy(listId);
+				setCopied(true);
+
+				setTimeout(() => {
+					setCopied(false);
+				}, 3000);
+			} catch (err) {
+				console.error('Copy failed:', err);
+			}
+		};
+
+		return (
+			<div>
+				<span>
+					List token: <em>{listId}</em>{' '}
+					<button onClick={handleCopyToClipboard}>
+						<FontAwesomeIcon icon={clipboard} />
```

Comment:
**question:** Is this icon currently setup for accessibility with a screen reader? I skimmed [the docs](https://fontawesome.com/docs/web/dig-deeper/accessibility#icons-used-as-semantic-elements) and I'm wondering if we're missing the `title` attribute to make it fully accessible... I may have missed something in my research though.

Reply:
Thank you for catching that! Just added the `title` attribute.